### ISSUE TITLE
No errors are triggered in develop

### ISSUE THREAD

**Issue by Bahamut66** [phpDocumentor/phpDocumentor2#1971]
For our project we are using phpDocumentor in the version of the develop. This is because we are building our project with PHP 7.1 and all the release version of phpDocumentor don’t support PHP 7.1 yet.
Judging from our tests we are under the impression that the develop version of phpDocumentor is compatible with PHP 7.1.
However, if we build a documentation using the develop version then phpDocumentor doesn’t log or document any errors.

## Expected Behavior
If we have a PHP class with missing DocBlocks, then we expect that the "run" process triggers some errors. 

## Actual Behavior
We have a PHP class with missing DocBlocks, but no errors are triggered during the build of the documentation.

## Steps to Reproduce the Problem

  1. create a php class with missing DocBlocks like this:
```
/**
 * This class manages tests for php documentor.
 *
 * Description of the class. (optional)
 */
class TestClass
{
    /**
      * The test constant.
      *
      * @var int
      */
    private const PRIVATE_CONST = 0;

    protected $testInt;

    public function testFunction(int $testOption)
    {
        return true;
    }
}
```

  2. run in terminal: vendor/bin/phpdoc -f ../src/Controller/TestClass.php -t php-doc
  3. no errors are written in terminal or in errors.html

## Your environment

* Version used: develop
* Install method: composer
* composer.json:
```
{
    "minimum-stability": "dev",
    "require": {
        "phpdocumentor/phpdocumentor": "dev-develop"
    }
}
```
* php version: 7.1
* Operating system and version (e.g. Ubuntu 16.04, Windows 7): Mac OS High Sierra (version 10.13.4)

**Comment by jaapio** [comment]
We decided to remove the validators since there are much better like phpstan that provide way more accurate errors than what we every could. Our reflection component is not parsing the method bodies so we won't be able to provide any useful errors except that there is something missing. 

I added your issue to our validation project. We might want to reintroduce the validation at some point but it won't be in the first version of phpdoc 3.
